envi/archs/aarch64/disasm.py

```diff
@@ -460,24 +706,32 @@ def p_test_branch_imm(opval, va):
         rt += meta_reg_bases[size]
     else:
         size = 8
+    if imm14 >> 13 & 0x1:   # Sign extension into 64 bits, handles negative values
+        imm14 = imm14 | 0xffffffffffffc000
     olist = (
-        A64RegOper(rt, va, size=size),
-        A64ImmOper(b5+b40, va=va),
-        A64ImmOper(imm14*0x100, va=va),
+        A64RegOper(rt, va, size=size), 
+        A64ImmOper((b5 * 0b100000) + b40, va=va), # Passes in decimal value
```

Comment:
this is better using shifts, not multiplication (for performance reasons)
```suggestion
        A64ImmOper((b5 << 5) + b40, va=va), # Passes in decimal value
```